example.env.create-cluster

```diff
@@ -0,0 +1,8 @@
+export AWS_PROFILE=moj-cp
+
+export KOPS_STATE_STORE=s3://cloud-platform-kops-state
```

Comment:
`KOPS_STATE_STORE` is an output of the cluster creation terraform config (`cloud-platform/`). We should be able to compute it: `KOPS_STATE_STORE="s3://$(terraform output kops_state_store)"`. It shouldn't change unless we decide to move the state elsewhere but it would remove the need to have it exported beforehand. Do you think it's worth changing?

Reply:
The previous cluster build process computed it. The problem is that I don't think it's possible to export an env. var from *within* a script, and have it available for sub-shells of that script. So, I specified that it should be in the environment.

Reply:
You should be able to `export` from the parent shell but not the other way around. In any case, I don't think the value will change anytime soon so we can leave it as-is.